docs/monitoring-usage.rst

```diff
@@ -278,3 +278,90 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation on `Custom Metrics`_ for more information.
+
+To write a data point to a custom metric, you must provide an instance of
+:class:`~gcloud.monitoring.metric.Metric` specifying the metric type as well as the values for
+the metric labels. You will need to have either created the metric descriptor earlier (see the
+`Metric Descriptors`_ section) or rely on metric type auto-creation (see `Auto-creation of
+custom metrics`_).
+
+You will also need to provide a :class:`~gcloud.monitoring.resource.Resource` instance specifying a
+monitored resource type as well as values for all of the monitored resource labels, except for
+``project_id``, which is ignored when it's included in writes to the API. A good choice is to use
+the underlying physical resource where your application code runs – e.g., a monitored resource
+type of ``gce_instance`` or ``aws_ec2_instance``. In some limited circumstances, such as when
+only a single process writes to the custom metric, you may choose to use the ``global`` monitored
+resource type.
+
+See `Monitored resource types`_ for a list of all monitored resource types available in
+Stackdriver Monitoring.
```

Comment:
Instead: "...for more information about particular monitored resource types."

This is to avoid the implication that the monitored resource types are available for writing custom metrics.